packages/conf-ida/conf-ida.0.1/files/find-ida.sh

```diff
@@ -33,8 +33,8 @@ find_command() {
 
 # can be very slow
 locate_linux() {
-    results=`locate -w -r 'idaq64$' | sort -n -r`
-
+    echo "Trying to locate IDA. It can take some time, consider to set IDA_PATH, e.g. IDA_PATH=/home/me/ida/idaq64"
```

Comment:
erm... the advice is not right, 
1) it should be a path to the folder where IDA lives
2) we should pass it through opam configuration
3) it is ungrammatical, should be 

"Searching for IDA installation on your machine, it might take some time. Consider passing the path via the opam configuration, e.g., `opam config --set ida-path /home/path/to/ida`